ex_smolvlm.py

```diff
@@ -1,37 +1,89 @@
 import requests
-import json
 import os
+import base64
+import logging
+import argparse
 
+logging.basicConfig(level=logging.INFO)
 
-ip_vllm = os.getenv('MY_URL','http://localhost:8000') #url declared as an env var, if empty, takes localhost:8000 by default
 
-model = os.getenv('MY_MODEL','HuggingfaceTB/SmolVLM-256M-Instruct') #model declared as an env var
+path = '/app/images'
+logging.info("Path")
+
+list_images = [file for file in os.listdir(path) if file.endswith((".jpg"))]
+logging.info("List of images")
```

Comment:
Crucial: You are logging some messages but without the content of relevant variables. In this case, I would output also the content of list_images.